core/embed/rust/src/ui/constants.rs

```diff
@@ -0,0 +1,13 @@
+//! Reexporting the `constant` module according to the
```

Comment:
please call this `constant` so that it's the same as model_*.constant

Reply:
Done in [902c3b2](https://github.com/trezor/trezor-firmware/pull/2262/commits/902c3b207836315adfb0e5aad4ffcd987c9e7cfd)